style.css

```diff
@@ -76,11 +80,158 @@ body {
     font-size: 16px;
     line-height: 24px;
     color: #7f8cff;
+    width: 100%;
   }
 
   .socialMedia {
     font-size: 22px;
     color: #505f79;
     font-weight: 500;
   }
+
+  .card {
+    border: 1px solid #dfe1e6;
+    padding: 16px;
+    border-radius: 16px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    flex-wrap: wrap;
+    width: 100%;
+  }
+
+  .work {
+    display: grid;
+    row-gap: 88px;
+    grid-template-columns: 100%;
+  }
+
+  img {
+    width: 100%;
+  }
+
+  h2 {
+    font-weight: 700;
+    text-align: left;
+    width: 100%;
+    margin-bottom: 0;
+  }
+
+  .title {
+    display: flex;
+    justify-content: start;
+    align-items: center;
+    height: 10%;
+    width: 100%;
+  }
+
+  .title h5 {
+    font-weight: 600;
+    color: #344563;
+    width: 22%;
+  }
+
+  .title_list {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    font-size: 13px;
+  }
+
+  .title_list li {
+    color: #c1c7d0;
+  }
+
+  .title_list li::before {
+    content: "\2022";
+    color: #c1c7d0;
+    font-weight: bolder;
+    display: inline-block;
+    width: 2em;
+  }
+
+  .tech_list {
+    display: flex;
+    justify-content: space-between;
+    width: 75%;
+    align-self: self-start;
+  }
+
+  .tech_list li {
+    color: #6070ff;
+    background-color: #ebebff;
+    border-radius: 8px;
+    padding: 4px 12px;
+  }
+
+  .btn_details {
+    color: #396df2;
+    align-self: self-start;
+    border: 1px solid #6070ff;
+    padding: 12px;
+    border-radius: 8px;
+    background-color: white;
+    font-weight: 500;
+    font-size: 17px;
+  }
+
+  .about_section {
+    padding-top: 20%;
+    padding-bottom: 20%;
+    display: flex;
+    flex-direction: column;
+  }
+
```

Comment:
- [ ] I think it would be better also to follow the Figma design by adjusting the Typography in the about section You may try to increase the size of the text to match the Figma design as much as possible ✨✨

- [ ]  It would be better also to add a ```border-top-right-radius:``` property to match the given Figma design and also to make a beautiful design as well 😎✨
  You may follow this [link](https://developer.mozilla.org/en-US/docs/Web/CSS/border-top-right-radius) for guidance on CSS border-radius 


**Your design** | **Figma design**
| --- | --- |
![image](https://user-images.githubusercontent.com/102757126/219435412-a202c9e3-e7e3-4e99-927d-9d16adb993ef.png) | ![image](https://user-images.githubusercontent.com/102757126/219436751-d4f21cfb-17ef-4571-823d-87fdb6d3dc0f.png)